src/bp/migrations/v12_21_0-1618260392-bot_content_type_update.ts

```diff
@@ -0,0 +1,22 @@
+import * as sdk from 'botpress/sdk'
+import { Migration } from 'core/migration'
+
+const migration: Migration = {
+  info: {
+    description: 'Adds video and audio content types to bot configs',
+    target: 'core',
+    type: 'config'
+  },
+  up: async ({ bp, configProvider }: sdk.ModuleMigrationOpts): Promise<sdk.MigrationResult> => {
+    const bots = await bp.bots.getAllBots()
+    for (const [botId, botConfig] of bots) {
+      configProvider.mergeBotConfig(botId, {
+        imports: { contentTypes: [...botConfig.imports.contentTypes, 'builtin_video', 'builtin_audio'] }
```

Comment:
If the content-types are already present in the bot's config, this migration will duplicate them. I know that usually, you shouldn't run the same migration twice, but maybe we should handle this edge case. What do you think?